homeassistant/components/lupusec/config_flow.py

```diff
@@ -0,0 +1,138 @@
+""""Config flow for Lupusec integration."""
+
+import ipaddress
+import logging
+import socket
+from typing import Any
+
+import lupupy
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import (
+    CONF_FRIENDLY_NAME,
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_USERNAME,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Optional(CONF_FRIENDLY_NAME): str,
```

Comment:
We still don't allow a custom name in the config flow :)

Reply:
Removed for config flow.